networkit/cpp/centrality/Betweenness.cpp

```diff
@@ -130,13 +130,11 @@ double Betweenness::maximum(){
     if (normalized) {
         return 1;
     }
-    double score;
-    count n = G.numberOfNodes();
-    if (G.isDirected()) {
-        score = (n-1)*(n-2);
-    } else {
-        score = (n-1)*(n-2)/2;
```

Comment:
The old code is fine here, since `(n-1)*(n-2)` is always even. In fact, the resulting `score` is an integer.

Reply:
But the return type is `double`.